### PR TITLE
edit spec to include dispense info and edit pipeline to make pr spec …

### PR DESCRIPTION
**Pull request by dan-jones-iw**
…visible

### CODE REVIEW

File: specification/electronic-prescription-service-api.yaml [hunk 1]
```diff
@@ -327,6 +331,64 @@ paths:
...
+  /Task/$release:
+    post:
+      operationId: release
+      summary: Send a prescription release request to EPS
+      description: |
+        ## Overview
+        Use this endpoint to send a prescription release request to EPS.
+
+        This will gather a set of prescriptions based on the information passed to the endpoint: 
+        * to gather prescription information for an entire pharmacy, send owner information
```
Comment (tjhmoore) [inline specification/electronic-prescription-service-api.yaml:343]: Do these make sense? Gather seems like they are getting but sending owner info seems like they are posting that they have downloaded them?

**Review by tjhmoore — COMMENTED** [review]
My comment may not be valid, let me know

**Review by tjhmoore — COMMENTED** [review]
Also, I see we have made the spec visible. Have we put in that the dispensing endpoint is just a spec at this point and cant be used?

File: specification/electronic-prescription-service-api.yaml [hunk 2]
```diff
@@ -339,9 +344,9 @@ paths:
         ## Overview
         Use this endpoint to send a prescription release request to EPS.
 
-        This will gather a set of prescriptions based on the information passed to the endpoint: 
-        * to gather prescription information for an entire pharmacy, send owner information
-        * to gather prescription information for a specific prescription, send the group identifier for that prescription
+        This will return a set of prescriptions based on the information passed to the endpoint:
```
Comment (SachinPath) [inline specification/electronic-prescription-service-api.yaml:347]: This returns a set of prescriptions based on the information passed to the endpoint:

File: specification/electronic-prescription-service-api.yaml [hunk 3]
```diff
@@ -339,9 +344,9 @@ paths:
         ## Overview
         Use this endpoint to send a prescription release request to EPS.
 
-        This will gather a set of prescriptions based on the information passed to the endpoint: 
-        * to gather prescription information for an entire pharmacy, send owner information
-        * to gather prescription information for a specific prescription, send the group identifier for that prescription
+        This will return a set of prescriptions based on the information passed to the endpoint: 
+        * to acquire prescription information for an entire pharmacy, send owner information
```
Comment (SachinPath) [inline specification/electronic-prescription-service-api.yaml:348]: * send owner information, to receive prescription information for an entire pharmacy.

File: specification/electronic-prescription-service-api.yaml [hunk 4]
```diff
@@ -339,9 +344,9 @@ paths:
...
+        * to acquire prescription information for a specific prescription, send the group identifier for that prescription
```
Comment (SachinPath) [inline specification/electronic-prescription-service-api.yaml:349]: * send the group identifier for a prescription, to receive prescription information for a specific prescription

**Review by ri-ch — COMMENTED** [review]
Is this definitely what a dispense request will look like? Presumably this is needed to release a prescription (or series of prescriptions) from Spine?

**Review by ri-ch — APPROVED** [review]

**Pull request merged**